platform/windows/os_windows.cpp

```diff
@@ -308,6 +311,83 @@ String OS_Windows::get_version() const {
 	return "";
 }
 
+Vector<String> OS_Windows::get_video_adapter_driver_info() const {
+	REFCLSID clsid = CLSID_WbemLocator; // Unmarshaler CLSID
+	REFIID uuid = IID_IWbemLocator; // Interface UUID
+	IWbemLocator *wbemLocator = NULL; // to get the services
+	IWbemServices *wbemServices = NULL; // to get the class
+	IWbemClassObject *pnpSDriverObject = NULL; // contains driver name, version, etc.
+	static String driver_name;
+	static String driver_version;
+
+	const String device_name = RenderingServer::get_singleton()->get_rendering_device()->get_device_name();
+	if (device_name.is_empty()) {
+		return Vector<String>();
+	}
+
+	CoInitialize(nullptr);
+
+	HRESULT hr = CoCreateInstance(clsid, NULL, CLSCTX_INPROC_SERVER, uuid, (LPVOID *)&wbemLocator);
+	if (hr != S_OK) {
+		return Vector<String>();
+	}
+
+	hr = wbemLocator->ConnectServer(L"root\\CIMV2", NULL, NULL, 0, NULL, 0, 0, &wbemServices);
+	SAFE_RELEASE(wbemLocator) // from now on, use `wbemServices`
+	if (hr != S_OK) {
+		SAFE_RELEASE(wbemServices)
+		return Vector<String>();
+	}
+
+	String object_name = vformat("Win32_PnPSignedDriver.DeviceName = \"%s\"", device_name);
+	// String to wchar_t*
+	char *data = const_cast<char *>(object_name.utf8().get_data());
+	int data_size = sizeof(data) / sizeof(data[0]);
+	int warg_size = MultiByteToWideChar(CP_UTF8, 0, data, data_size, nullptr, 0);
+	wchar_t *object_name_wide = new wchar_t[warg_size];
+	MultiByteToWideChar(CP_UTF8, 0, data, data_size, object_name_wide, warg_size);
```

Comment:
In case of Windows, you can convert `String` to `wchar_t *` with built-in `string.utf16().get_data()` (cast result to `LPCWSTR` or whatever API takes).